isis/src/base/apps/phocube/main.cpp

```diff
@@ -447,15 +441,35 @@ void phocube(Buffer &out) {
             index += 64 * 64;
           }
         }
-        
       }
-      // Trim outerspace
+
+      // Trim outer space except last two bands when RA and declination are selected
+      // We always skip two because RA and declination are selected together.
       else {
-        for(int b = (skipDN) ? 1 : 0; b < nbands; b++) {
+        int bandsToProcess;
+        if (ra && declination) {
+          bandsToProcess = nbands - 2;
```

Comment:
Are ra and declination required to be together by the app xml?

Reply:
Yes.